sites/website/src/docs/design-systems/using-the-design-system.md

```diff
@@ -0,0 +1,130 @@
+---
+id: using-the-design-system
+title: Using the FAST Design System 
+sidebar_label: Using the Design System
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/using-the-design-system.md
+---
+
+This section goes into further detail on how the Design System can be used. It will use [FAST Frame](/docs/design-systems/fast-frame) Design System in the examples but none are *specific* to [FAST Frame](/docs/design-systems/fast-frame); use these features with your own Design System if you're not using [FAST Frame](/docs/design-systems/fast-frame).
+
+### Setting Design System Properties
+Design System properties can always be set by simple property assignment in JavaScript. Simply obtain a reference to the Custom Element instance and assign the property.
```

Comment:
omit "simple"

Reply:
"Design System properties can be set by property assignment in JavaScript by obtaining a reference to the Custom Element instance and assign the property."